conjur/logic/credential_provider/credential_store_factory.py

```diff
@@ -51,16 +43,13 @@ def create_credential_store(self) -> CredentialsStoreInterface:
         self._log_netrc_warning()
         return FileCredentialsProvider(), DEFAULT_NETRC_FILE
 
-    def override_usage(self, provider_type: CredentialProviderTypes):
+    def _log_netrc_warning(self):
         """
-        if set to 'NETRC' create_credential_store will always return FileCredentialsProvider
-        if set to 'KEYSTORE' create_credential_store will always return KeystoreCredentialsProvider
+        Method logging the fact we are using insecure CredentialProvider.
+        This will write to log for every cli run
```

Comment:
```suggestion
        This will be displayed to the user as a warning on every CLI run
```

Will also be prompted to the user not only in the logs